meson.build

```diff
@@ -0,0 +1,14 @@
+project('gtk-layer-shell', ['c'], version: '0.1.0', license: 'MIT',
+	default_options: ['c_std=c99', 'warning_level=2'])
+
+gtk = dependency('gtk+-wayland-3.0', version: '>=3.22.0')
+wayland_client = dependency('wayland-client', version: '>=1.10.0')
+wayland_scanner = dependency('wayland-scanner', version: '>=1.10.0', required: false)
```

Comment:
Wayland scanner is required as a build dependency. It looks like if this doesn't find wayland-scanner, it looks like it tries something else below. Can you indicate in a comment that wayland-scanner *is* required, but it has another way of looking for it if this fails?